.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## Follow these steps before creating a PR and then remove this text
+
+Does this PR prepare the schema for a public release?
+
+If YES, then:
+1. Remove the suffix "-alpha" from the version string in `conf.py` and `namespace.yaml`
+2. Update the date in the release notes
+3. Follow release instructions
+
+If NO, then:
+  Is the current schema version on "master" a public release?
+
+  If YES, then:
+  1. Update the version string in `conf.py` and `namespace.yaml` to the next version with the suffix "-alpha"
+  2. Add a new section in the release notes for the new version with the date "Upcoming"
+  3. Add release notes for the PR
+
+  If NO, then:
+  1. Add release notes for the PR
+
+See https://hdmf-common-schema.readthedocs.io/en/latest/software_process.html for more details.
```

Comment:
This may be good to add as a checklist to the issue template.

Reply:
Will do.

Reply:
Do you mean turn this list into a checklist in the PR template? I don't think it makes sense to add this to the Bug Report / Feature Request template...

Reply:
How about a "Make a release" PR?

Reply:
It would be great to have separate templates for normal PR and "Make a release" PR. Although GitHub allows you to make multiple PR templates, you can't pick between templates when making a PR. Instead, you have to specify the template by adding 'template=prepare-release.md' to the URL. That is inconvenient, but let's give it a try.

Reply:
Hmm. Actually, if you have multiple PR templates, then GitHub does not pick one to be the default, so all PRs must append to the URL in order to get a template. That doesn't seem worth it.